zigpy_zigate/zigbee/application.py

```diff
@@ -153,6 +155,8 @@ def zigate_callback_handler(self, msg, response, lqi):
         elif msg == 0x8702:  # APS Data confirm Fail
             LOGGER.debug('APS Data confirm Fail %s %s', response[4], response[0])
             self._handle_frame_failure(response[4], response[0])
+        elif msg == 0x9999:  # ZCL event
+            LOGGER.debug('ZCL application started %s', response[0])
```

Comment:
This debug message is not really understandable in my opinion, maybe something like "Extended error code %s" is better ?

Reply:
I did not even knew it was an error, I thought it was a normal application response

Reply:
Logging an Error as a Debug looks strange

Reply:
Do you have somewhere a documentation that could help me understand this 0x9999 event and what it means?

I have a bunch of them at each restart of home-assistant, I thought it was normal.

Is it ok if I raise this to a warning level @pipiche38?